pipeline.py

```diff
@@ -72,31 +89,79 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
+        self.corpus_manager = corpus_manager
         pass
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for article_item in self.corpus_manager.get_articles().values():
+            article_text = article_item.get_raw_text()
+            article_text = article_text.replace('-\n', '')
+            for symbol in ['\n', '\r']:
+                article_text = article_text.replace(symbol, ' ')
+            re_tokens = re.findall(r'[а-яё_]+-?[а-яё_]*', article_text, flags=re.IGNORECASE)
+            cleaned_text = ' '.join(re_tokens)
+            article_item.save_as(cleaned_text.lower(), 'cleaned')
+            self._process(cleaned_text)
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        m_tokens_list = []
+        analyzed_text_mystem = Mystem().analyze(raw_text)
+        for analyzed_word in analyzed_text_mystem:
+            if analyzed_word['text'] == ' ':
+                continue
+            original_word = analyzed_word['text']
+            token = MorphologicalToken(original_word)
```

Comment:
what is the reason to create an instance before making the checks? if checks fail, this instance is not used